src/main/steps/applicant1/withdrawing-your-application/content.ts

```diff
@@ -1,15 +1,51 @@
+import { Checkbox } from '../../../app/case/case';
 import { TranslationFn } from '../../../app/controller/GetController';
 import { CommonContent } from '../../common/common.content';
 
-const en = ({ isDivorce, partner, isJointApplication }: CommonContent) => ({
+const en = ({ isDivorce, partner }: CommonContent) => ({
   title: `Withdrawing your ${isDivorce ? 'divorce application' : 'application to end your civil partnership'}`,
-  line1: `You have said you do not want to continue with your ${
-    isDivorce ? 'divorce' : 'application to end your civil partnership'
-  }. If you want to withdraw your application then you${
-    isJointApplication ? ` and your ${partner}` : ''
-  } need to fill out a separate <a class="govuk-link" href="https://www.gov.uk/government/publications/form-d11-application-notice">D11 form</a> and send it to the court. Details of where to send it are on the form.`,
-  line2:
-    'If you need help then you can contact the court using the details below. The support staff cannot give you legal advice.',
+  sole: {
+    line1: `You have said you do not want to continue with your ${
+      isDivorce ? 'divorce' : 'application to end your civil partnership'
+    }. If you want to withdraw your application then you need to fill out a separate
+    <a class="govuk-link" href="https://www.gov.uk/government/publications/form-d11-application-notice" target="_blank">D11 form</a>
```

Comment:
I know this is mid-sentence but I still think it's better off being in the template file

Reply:
@benouaer As in move the whole line into the template?

Reply:
no, just the anchor tag

Reply:
we can break line1 text into separate bits to do that

Reply:
@benouaer since there's a lot of mid line links in this file, I'll merge this pr and then raise a new one to update them